src/modules/DNS.rb

```diff
@@ -479,6 +482,21 @@ def Import(settings)
 
       @nameservers = Builtins.eval(Ops.get_list(settings, "nameservers", []))
       @searchlist = Builtins.eval(Ops.get_list(settings, "searchlist", []))
+
+      # check for AY unsupported scenarios, the name servers and the search domains
+      # are written in the 2nd stage, if is disabled then it cannot work (bsc#1046198)
```

Comment:
Yes, I agree that it's not a real bug. But on the other hand nobody reads the documentation, if we print an error we might report false bug reports...

The tricky part is that the profile validates, and unless you know this limitation (or read the documentation) there is no way to find out that the profile is not correct.